echo-webhooks/src/test/groovy/com/netflix/spinnaker/echo/controllers/WebhooksControllerSpec.groovy

```diff
@@ -334,4 +361,163 @@ class WebhooksControllerSpec extends Specification {
     response.eventProcessed == true
     response.eventId == event.eventId
   }
+
+  void 'handles Github Webhook Event'() {
+    def event
+
+    given:
+    WebhooksController controller = new WebhooksController(mapper: new ObjectMapper(), scmWebhookHandler: scmWebhookHandler)
+    controller.propagator = Mock(EventPropagator)
+    controller.artifactExtractor = Mock(ArtifactExtractor)
+    controller.artifactExtractor.extractArtifacts(_, _, _) >> []
+
+    when:
+    def response = controller.forwardEvent(
+      "git",
+      "github",
+      """{
+  "ref": "refs/heads/simple-tag",
+  "before": "a10867b14bb761a232cd80139fbd4c0d33264240",
+  "after": "0000000000000000000000000000000000000000",
+  "repository": {
+    "name": "Hello-World",
+    "owner": {
+      "name": "Codertocat"
+    }
+  }
+}
+""",
```

Comment:
http://mrhaki.blogspot.com/2010/06/groovy-goodness-strip-leading-spaces_20.html

Reply:
Seems like this actually works fine without `stripMargin` so I just moved these over.

Reply:
Seems like this actually works fine without `stripMargin` so I just moved these over.